src/locales/pl/PAGES/START.json

```diff
@@ -0,0 +1,29 @@
+{
+    "TITLE": "WCAG-EM. Generator raportów",
+    "SUBTITLE": "Przegląd",
+    "INTRO_HD": "Co robi to narzędzie",
```

Comment:
```suggestion
    "INTRO_HD": "Do czego służy to narzędzie?",
```

Reply:
Zasady pisowni pozwalają na pominięcie znaku zapytania w nagłówku (śródtytule)

Reply:
Zasady pisowni to jedna rzecz, ale wygoda czytania i przyjemność korzystania to rzecz druga. ;) Ale oczywiście zgadzam się, jeśli chcemy trzymać się sztywnej jak kij od szczotki językowej konwencji pism urzędowych ;)